appserver/distributions/pom.xml

```diff
@@ -104,88 +108,66 @@
-                            <id>default-zip</id>
-                            <phase>package</phase>
+                            <id>create-domain</id>
+                            <phase>prepare-package</phase>
+                            <goals>
+                                <goal>exec</goal>
+                            </goals>
                             <configuration>
-                                <excludes>**/.gitkeep_empty_dir</excludes>
-                                <duplicate>preserve</duplicate>
-                                <filesets>
-                                     <fileset>
-                                        <fileMode>600</fileMode>
-                                        <dir>${stage.dir}</dir>
-                                        <includes>**/*.jks</includes>
-                                        <includes>**/admin-keyfile</includes>
-                                        <includes>**/domain-passwords</includes>
-                                        <includes>**/keyfile</includes>
-                                    </fileset>
-                                    <fileset>
-                                        <fileMode>640</fileMode>
-                                        <dir>${stage.dir}</dir>
-                                        <includes>**/bin/*.1m</includes>
-                                        <includes>**/bin/*.jar</includes>
-                                    </fileset>
-                                    <fileset>
-                                        <dir>${stage.dir}</dir>
-                                        <fileMode>750</fileMode>
-                                        <includes>**/bin/*</includes>
-                                        <includes>**/bin/*.bat</includes>
-                                        <includes>**/config/*.bat</includes>
-                                        <includes>**/config/*.conf</includes>
-                                        <includes>**/lib/nadmin*</includes>
-                                        <includes>**/imq</includes>
-                                        <includes>**/imqinit</includes>
-                                        <excludes>**/bin/*.1m</excludes>
-                                        <excludes>**/bin/*.jar</excludes>
-                                    </fileset>
-                                    <fileset>
-                                        <fileMode>640</fileMode>
-                                        <dir>${stage.dir}</dir>
-                                        <excludes>**/bin/*</excludes>
-                                        <excludes>**/bin/*.bat</excludes>
-                                        <excludes>**/lib/nadmin*</excludes>
-                                    </fileset>
-                                </filesets>
+                                <executable>${stage.dir}/${install.dir.name}/bin/asadmin</executable>
```

Comment:
Failed on Windows

Reply:
I hate Windows
- git clone failed, too long file paths, resolved by git config
- gjule tests failed, `\r`. For some reason `-Dfile.separator='\n'` doesn't have any effect. Test skipped.
- exousia 2.1.1 is missing in maven central ... reported to @arjantijms 

I will get there, can you give me some hint how to avoid the `'\r'` issue?

Reply:
@dmatej, instead of `split("\n")` you sould use either `System.lineSeparator()` or `\\r?\\n` pattern in place of `\n`. In compiled patterns use same pattern `\\r?\\n` instead of `\\n`. E.g. `\\n\\n` would be translated to `(\\r?\\n){2}` and so on

Hope help :)

Reply:
Hmm, I hoped I can somehow just force windows to switch to unix lines, but it seems I really have to do this.